youtube_dl/extractor/vgtv.py

```diff
@@ -11,16 +11,19 @@
 
 
 class VGTVIE(InfoExtractor):
-    IE_DESC = 'VGTV and BTTV'
+    IE_DESC = 'VGTV, BTTV, FTV, Aftenposten and Aftonbladet'
     _VALID_URL = r'''(?x)
-                    (?:
-                        vgtv:|
```

Comment:
i can make it compatible but then it can match urls that are not actualy available in the site.